packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -50,12 +50,12 @@ const bottomMenuItems: SidebarMenuItem[] = [
   {
     icon: <ListIcon Icon={TerminalIcon} />,
     title: 'Changelog',
-    path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}sources/daily_updates `,
+    path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}sources/daily_updates`,
   },
   {
     icon: <ListIcon Icon={FeedbackIcon} />,
     title: 'Feedback',
-    path: 'https://it057218.typeform.com/to/S9p9SVNI',
+    path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}feedback`,
```

Comment:
`NEXT_PUBLIC_WEBAPP_URL` is actually translated to `https://app.daily.dev`
And the original link is: `https://daily.dev/feedback`.
It will be better to add it hard coded :)